SU-HSDO/suhumsci
# SHS-5572: The arrows on a Spotlight (with no image) are displaying incorrectly on mobile

# READY FOR REVIEW

## Summary
Fix arrows position on mobile for Spotlight (with no image)

## Need Review By (Date)
04/19

## Urgency
medium

## Steps to Test
1. Create a Flexible Page on Colorful and Traditional and add a component of Spotlight without adding images and add another with images
2. Confirm the position of the arrow in all viewports (specially on mobile) in both components look better 

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
